packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -777,36 +775,71 @@ export default class Meetings extends WebexPlugin {
             // @ts-ignore
             `Meetings:index#register --> INFO, Device registered ${this.webex.internal.device.url}`
           )
-        )
-        // @ts-ignore
-        .then(() => this.webex.internal.mercury.connect()),
+        ),
       MeetingsUtil.checkH264Support.call(this),
-    ])
-      .then(() => {
-        this.listenForEvents();
-        Trigger.trigger(
-          this,
-          {
-            file: 'meetings',
-            function: 'register',
-          },
-          EVENT_TRIGGERS.MEETINGS_REGISTERED
-        );
-        this.registered = true;
-        Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_SUCCESS);
-      })
-      .catch((error) => {
-        LoggerProxy.logger.error(
-          `Meetings:index#register --> ERROR, Unable to register, ${error.message}`
-        );
-
-        Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_FAILED, {
-          reason: error.message,
-          stack: error.stack,
-        });
+    ]).catch((error) => {
+      LoggerProxy.logger.error(
+        `Meetings:index#register --> ERROR, Unable to preregister, ${error.message}`
+      );
 
-        return Promise.reject(error);
+      Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_FAILED, {
+        reason: error.message,
+        stack: error.stack,
       });
+
+      return Promise.reject(error);
+    });
+  }
+
+  /**
+   * Explicitly sets up the meetings plugin by registering
+   * the device, connecting to mercury, and listening for locus events.
```

Comment:
Q: Is this still a valid description?

Reply:
I updated the jsdoc with more information